docs/daemon/validator.mdx

```diff
@@ -0,0 +1,36 @@
+
+There are 2 ways to become a validator:
+1. Pre-establish them in the genesis file before starting the Kwil network.
+2. Going through a federated consensus-based join process.
+
+## Genesis File
+The Kwil network can be deployed with a pre-determined set of validators by including them in the genesis file. These genesis validators must be specified under the section `"validators"` in the genesis file. For more information, refer to the [genesis validators guide](./config/genesis#genesis-validators)
+
+## Federated Consensus-Based Join 
+
+If a node wants to become a validator post-network deployment, it can do so by undergoing a PBFT consensus-based approval process. The process of joining the network as a validator through federated consensus involves the following steps:
+
+1. A node that wishes to become a validator must issue a join request transaction signed by its private key. Join requests can be issued using the `kwil-admin` command line tool. For more information, refer to the [join request guide](../../admin/validators.mdx#join).
+
+2. Upon receiving the join request, existing validators must determine if the node should be granted validator status.  Validator nodes that decides to approve must issue an approval request for the joiner, signed by the validator's private key. This approval request is broadcasted as a blockchain transaction. Refer to the [approval request guide](../../admin/validators.mdx#approve) for more information on issuing approval requests using `kwil-admin` tool.
+
+3. The join requests remain active for the duration of `join_expiry` defined in the genesis file. The join request will expire if 2/3rd of the current validators don't approve the join request within this duration. The status of the Join request can be queried using the `kwil-admin` command line tools. Refer to the [join request status guide](../../admin/validators.mdx#show-join-request-status) for more information.
+
+4. When each validator node receives a super-majority (2/3rd) of approvals from the current validators, they include the joiner node into a set of validator updates sent to the CometBFT core. CometBFT then reaches a consensus on these ValidatorSet updates and, if agreed upon, elevates the joiner node to validator status. The joiner node can then begin participating in the consensus decisions from the next round.
+
+### Leave Validator Status
+
+If a validator node chooses to relinquish its status as a validator, it can initiate a leave request transaction signed by its private key. Once the transaction is mined, CometBFT updates the validator power to 0, effectively demoting the node from the validator status. Starting next round, the node will no longer be able to participate in the consensus decisions.
```

Comment:
"starting next block", since the reader presumably does not have knowledge of CometBFT's rounds